control.lua

```diff
@@ -108,6 +110,7 @@ local function onInit()
     global.world = {}
 
     world = global.world
+    minDiffuse = game.map_settings.pollution.min_to_diffuse
```

Comment:
why create this as a separate variable and not use the world global that was already created?

Reply:
I don't understand. Your global variable is for your data while minDiffuse only takes game map settings property. The idea is that this game setting may change between games (I saw some mods to change it - abnormal situation, because it mustn't change after map creation, but some people create mods to change it).
Please change it according to your vision. The only idea here is to take that important map setting value instead of hard coded value = 15.